oaiharvest/metadata.py

```diff
@@ -45,9 +45,10 @@ def writeMetadata(self, metadata_prefix, element, metadata):
 class XMLMetadataReader(object):
     """Really simple MetadataReader to serialize metadata to pretty XML."""
     def __call__(self, metadata_element):
-        return '\n'.join([tostring(rec_element,
-                                   method="xml",
-                                   pretty_print=True
-                                   )
-                          for rec_element
-                          in metadata_element])
+        # Six call fixes 'sequence item 0: expected str instance, bytes found'
+        return ('\n' if six.PY2 else b'\n').join(
```

Comment:
I'm testing with Python 3.5.2 on Ubuntu 16.04.2 - After fixing other bugs I got the error message
```
ERROR Fix sequence item 0: expected str instance, bytes found
```
I don't really know why it returned a `bytes`, but for some reason it did for at least one item in the list...